tests/ComposerCommandsTest.php

```diff
@@ -19,6 +19,12 @@ protected function setUp(): void
         parent::setUp();
         $this->startServer();
         $this->composer('require', 'php-tuf/composer-integration');
+
+        // Enable TUF protection at the command line so we know it works.
+        $this->assertStringContainsString(
+          "http://localhost:8080 is now protected by TUF.",
+          $this->composer('tuf:protect', 'fixture')->getOutput(),
+        );
```

Comment:
Should we also the file was update correctly and that it will fail for a non-composer repo?

Reply:
> Should we also the file was update correctly 

No need. If it wasn't enabled, the rest of the test would fail because TUF support would be disengaged.

> that it will fail for a non-composer repo?

I could test this, sure.

Reply:
Okay, I added a test to prove that the command fails if you try to use it on a not-Composer repository.